requirements.txt

```diff
@@ -1,19 +1,3 @@
 torch>=1.7  # TODO: regenerate weights with lower PT version
 torchmetrics
-torchvision>=0.8  # TODO: lower to 0.7 after PT 1.6
 pytorch_lightning==1.3.0rc1
-lightning-bolts==0.3.2  # TODO: align with proper release
-PyYAML>=5.1
-Pillow>=7.2
-transformers>=4.0
-pytorch-tabnet==3.1
-datasets>=1.2, <1.3
-pandas>=1.1
-scikit-learn>=0.24
-numpy  # comes with 3rd-party dependency
-tqdm  # comes with 3rd-party dependency
-rouge-score>=0.0.4
-sentencepiece>=0.1.95
-filelock  # comes with 3rd-party dependency
-pycocotools>=2.0.2 ; python_version >= "3.7"
-kornia>=0.5.0
```

Comment:
Now that these are not automatically installed, we should if each package is available every time we try to import it and print the appropriate `pip install` command to include it.

Reply:
I'm suggesting a simpler way of importing optional packages in #102.
```python
with ContextManager() as cm:
    from kornia import ...

cm.check()  # Here, the code block will get simple.
```

I'd love to know your thoughts on it :]